src/PackageDependencyViewExtension/PackageDependencyView.xaml

```diff
@@ -86,5 +86,51 @@
                 </TextBlock>
             </Button>
         </Grid>-->
+
+        <!-- Template for the feedback banner -->
+        <DockPanel Name="Feedback" Margin="0,800,0,40" Grid.Row="1">
+            <StatusBar Background="CornflowerBlue" DockPanel.Dock="Bottom" Height="55">
```

Comment:
as @ColinDayOrg says - it would be good if the colors here referenced the modern themes where possible so that in the future it's simpler for users to remap them.

Reply:
Yes, added this to DynamoColorsAndBrushes.xaml file.